client/src/main/java/com/github/gotify/client/StringUtil.java

```diff
@@ -13,7 +13,7 @@
 
 package com.github.gotify.client;
 
-@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaClientCodegen", date = "2020-06-24T18:39:01.386+02:00")
+//@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaClientCodegen", date = "2020-06-24T18:39:01.386+02:00")
```

Comment:
You can probably remove them for now and we can disable the annotation in the client generation, when a new client will be generated. https://github.com/swagger-api/swagger-codegen/issues/2834